federation/cluster/common.sh

```diff
@@ -135,6 +135,30 @@ function create-federation-api-objects {
 	    fi
 	    sleep 5
 	done
+
+	# Poll until DNS record for federation-apiserver is propagated
```

Comment:
kubectl will fail because the DNS name for the ELB could not be resolved. I guess we could soak that up by retrying kubectl for a few minutes, but this seems more explicit.